frontend/src/metabase/metadata/components/SyncTableSchemaButton/SyncTableSchemaButton.tsx

```diff
@@ -0,0 +1,38 @@
+import { useEffect, useState } from "react";
+import { t } from "ttag";
+
+import { useSyncTableSchemaMutation } from "metabase/api";
+import { useToast } from "metabase/common/hooks";
+import { Button } from "metabase/ui";
+import type { TableId } from "metabase-types/api";
+
+export function SyncTableSchemaButton({ tableId }: { tableId: TableId }) {
+  const [started, setStarted] = useState(0);
```

Comment:
Let's keep this no different than in other similar buttons (`DiscardFieldValuesButton`, `DiscardTableFieldValuesButton`, etc.)